codegen/files_to_generate/emosqp_mex.c

```diff
@@ -299,6 +299,34 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
         return;
     }
 
+    // update warm_start
+    if (!strcmp("update_warm_start", cmd)) {
+
+        // Fill l, u
+        const mxArray *x = prhs[1];
+        const mxArray *y = prhs[2];
+
+        // Copy vectors to ensure they are cast as c_float
+        c_float *x_vec;
+        c_float *y_vec;
+        if(!mxIsEmpty(x)){
+            x_vec = copyToCfloatVector(mxGetPr(x), (&workspace)->data->m);
+        }
+        if(!mxIsEmpty(y)){
+            y_vec = copyToCfloatVector(mxGetPr(y), (&workspace)->data->m);
+        }
+
+        if(!mxIsEmpty(y)){
```

Comment:
Is there a reason the call to warm starting is guarded against y being non-empty? As another note, with the update above to make this function callable from the wrapper, it should be guaranteed to get both an x and y to warm start.